### DISCUSSION TITLE
Unable to Set Camera Inside 3D Model in ModelViewer

### DISCUSSION THREAD

**Discussion by akssieg** [google/model-viewer#4836] [Q&A] [answered]
### Description
I loaded a 3D apartment model into ModelViewer. When using simple Three.js, I can set the camera positions and move the camera inside the 3D model to view the interiors. However, this does not seem possible with ModelViewer.

### Additional Information
I have the camera positions in both Cartesian and spherical coordinates.
ModelViewer does not accept the spherical coordinates that place the camera inside the model.

### Attached Image
<img width="724" alt="image" src="https://github.com/google/model-viewer/assets/32209750/55308fe0-53a7-4baa-ab65-cdffd797caca">

### Request
Please provide guidance on where refinements are needed in the ModelViewer source code to allow the camera to move inside the 3D model. Specifically, how to set the camera positions correctly for this purpose.

Thank you!
@elalish @jsantell

**Comment by elalish** [comment, marked as answer]
This is, in fact possible - you just have to set the camera limits before you set the camera itself: https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-attributes-minCameraOrbit

For inside-out viewing, I'd recommend `min-camera-orbit="auto auto 0m" camera-orbit="auto auto 0m" orbit-sensitivity="-1"`
